Category-Colors/category-colors
# Add prettier

# Description
Issue #24 
- Adding Prettier to format our code using a pre-commit hook

# QA
- Checkout this branch
- Run `npm install`
- Run `npm pkg set scripts.prepare="husky install"`
- Run `npx husky add .husky/pre-commit "npx lint-staged"`
- The next time you run `git commit` the hook will run prettier and format all files that have not been formatted properly. You'll see something like the below:
![Screenshot 2023-06-27 at 3 33 06 PM](https://github.com/Category-Colors/category-colors/assets/2035194/378668c1-b14a-4050-be76-f55721a61c40)
